index.js

```diff
@@ -107,8 +124,22 @@ Use the hungryDog function and feeding requirements below to do the following:
   NOTE: If done correctly, a weight of 15 lbs and age of 1 year would return 0.44999999999999996
 */  
 
-function hungryDog(/*add your code here*/){
-    /*add your code here*/
+function hungryDog(weight, age){
+  if (age>=1&&weight<=5){
+    return .05*weight;
+  } else if (age>=1&&(weight>=6&&weight<=10)){
+    return .04*weight;
+  } else if (age>=1&&(weight>=11&&weight<=15)){
+    return .03*weight;
+  } else if (age>=1&&(weight>15)){
+    return .02*weight;
```

Comment:
Lines 128- 135 all have the condition `if age >= 1`... While your code may work, it may be hard to read so I would group those into one and then nest the weight conditions in. 

```suggestion
  if ( age >= 1 ){
      if (weight >= 5) {
         return .05 * weight
      } else if (other weight) {
         return etc...
      } 
       // all other weight conditions for this age can be nested into this if statement
   
    } else if ( age >= .16 && age <= .33 ){
    return .10*weight;
  } else if ( age > .33 && age <= .58){
    return .05*weight;
  } else if ( age > .58 && age <= .99 ){
    return .04*weight;
  }
```
Also, just a friendly suggestion install this extension on your vs code editor to format your code. 
---> [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)